packages/ui-components/.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "attach",
+            "name": "Attach",
```

Comment:
Can we give this debug config a more descriptive name, maybe "Attach to Jest test"? vscode allows comments in these json files, so we may also want to include detailed inline instructions on exactly how to spin this debug job up. I assume it's:

- `jlpm test:debug:watch`
- run this debug job in vscode

Reply:
I like it, but as a single sweep after the migration is complete. Mind opening an issue?

Reply:
done: #8327

Reply:
Thanks!